### PR TITLE
chore: jq update messaging

### PR DESCRIPTION
**Pull request by hamidzr**
## Description

- job q messaging refactor
- report job's : user, name

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: master/internal/resourcemanagers/resource_pool.go [hunk 1]
```diff
@@ -281,6 +285,12 @@ func (rp *ResourcePool) Receive(ctx *actor.Context) error {
 			}
 			rp.sendScalingInfo(ctx)
 		}
+		// jobQ := rp.scheduler.JobQInfo(rp) // OPTIMIZE: integrate into Schedule() call
```
Comment (hamidzr) [inline master/internal/resourcemanagers/resource_pool.go:288]: this is how we could move forward to cache jobQ in jobs actor

**Comment by hamidzr** [comment]
pr for updated config usage https://github.com/determined-ai/determined/pull/3175

**Review by eecsliu — APPROVED** [review]
looks pretty good to me!

File: master/internal/command/command.go [hunk 2]
```diff
@@ -375,14 +406,42 @@ func (c *command) refreshAllocationState(ctx *actor.Context) task.AllocationStat
...
+	if priority := c.Config.Resources.Priority; priority != nil {
+		j.Priority = int32(*priority)
+	} else {
+		j.Priority = 42
+	}
+
+	// TODO make config its own package so we can bring in the config
+	// to resolve preemption, priority, and weight?
+
+	job.UpdateJobQInfo(&j, c.rmJobInfo)
```
[the remark this answers is not in the document]
Comment (hamidzr) [reply master/internal/command/command.go:439]: makes sense. it could also read from the jobs actor if we cache the q info in there.

**Comment by hamidzr** [comment]
thanks for the review. 
I removed the temp webui commit

**Pull request merged**